src/shared/settings.d.ts

```diff
@@ -20,7 +20,7 @@ export interface Settings {
     appBadge?: boolean;
     disableMinSize?: boolean;
     clickTrayToShowHide?: boolean;
-    customTitleBar?: boolean;
```

Comment:
when changing settings names, old settings should be migrated to the new key so users won't have their choice reset. you will likely have to do this in the main process so it applies immediately